### ISSUE TITLE
How i can add "consumes"?

### ISSUE THREAD

**Issue by Gemorroj** [nelmio/NelmioApiDocBundle#1302]
```php
    /**
     * @Route("/api/private/categories/add", methods={"POST"}, defaults={"_format": "json"})
     * @Operation(
     *     consumes={"application/x-www-form-urlencoded"},
     * )
     * @SWG\Response(
     *     response=200,
     *     description="Добавление категории",
     *     @Model(type=Category::class, groups={"category"}))
     * )
     * @SWG\Response(
     *     response=400,
     *     description="Ошибка валидации"
     * )
     * @SWG\Parameter(
     *     name="categoryName",
     *     in="formData",
     *     type="string",
     *     description="Имя категории",
     *     required=true
     * )
     * @Security(name="Bearer")
     */
```

i have error: `User Notice: Unable to merge @SWG\Post() in \PrivateController->addCategoryAction()`
I do not use `FosRestBundle`

**Comment by chregu** [comment]
Exactly the same question/issue here.
Also with eg `@Operation(operationId="myownId")`

**Comment by GuilhemN** [comment]
This is fixed by #1321 :)
